application/src/main/resources/permissions.json

```diff
@@ -8,15 +8,5 @@
       "hono-client": [ "WRITE" ],
       "rest-adapter": [ "WRITE" ],
       "mqtt-adapter": [ "WRITE" ]
-  },
```

Comment:
why do we need to keep this file at all?
The permissions are managed by Auth Server ...